[DOCS] Add custom fields to cases (#4070)
elastic/security-docs: docs/cases/cases-manage.asciidoc

image::images/cases-ui-open.png[Shows an open case]
// NOTE: This is an autogenerated screenshot. Do not edit it directly.

[float]
[[cases-ui-custom-fields]]
== Add custom fields

preview::[]

You can add optional and required fields for customized case collaboration.

. Go to *Cases* and click *Settings*.
+
--
[role="screenshot"]
image::images/cases-custom-field-settings.png[View custom fields in case settings]
// NOTE: This is an autogenerated screenshot. Do not edit it directly.

NOTE: To view and change case settings, you must have the appropriate {kib} feature privileges. Refer to <<case-permissions>>.
--

. In the *Custom fields* section, click *Add field*.
+
--
[role="screenshot"]
image::images/cases-add-custom-field.png[Add a custom field in case settings]
// NOTE: This is an autogenerated screenshot. Do not edit it directly.
--

. Enter a field label.

. Choose a field type: text or toggle.

> . If you want the text field to be mandatory in all cases, select *Make this field required*.

Reviewer: ```suggestion
. (Optional) To make the text field mandatory, select *Make this field required*.
```

. Click *Save field*.

> You can subsequently remove or edit custom fields on the *Settings* page.

Reviewer: I kinda want to bring the next three lines together so they look less disjointed. Here's an option:

```suggestion
After you create custom fields, they're added to all new and existing cases. Existing cases have null values for new text fields until you set them in. To manage custom fields, go the *Settings* page and remove or edit them.
```

Author: Fixed in f0bb1b1e12afa0e2583223dce275bf58fce07103, thanks!
